.gitignore

```diff
@@ -0,0 +1,2 @@
+TenantFile/Api/tenant-file-12264437b25e.json
```

Comment:
What was this for? Was this file the side effect of something?

Also, the ionide file should probably be added.

Reply:
That file was created for holding Google Cloud authorization as per one of the methods, but it ended up not being needed. Deleting it from my local repository and from the .gitignore for the next commit.